app/models/manageiq/providers/azure/cloud_manager/refresh_parser.rb

```diff
@@ -46,6 +47,10 @@ def ems_inv_to_hashes
 
       private
 
+      def get_network_interfaces
+        @data[:network_interfaces] = gather_data_for_this_region(@nis)
```

Comment:
@djberg96 
Looks like the @data hash contains the NIC objects returned from the armrest gem? We store our own hashes here in the process_collection method. Can you do a double check?